downloads/platform.md

```diff
@@ -65,10 +65,9 @@ To start running Julia from the Terminal, you can do the following:
 
 Navigate to `/usr/local/bin` and remove the `julia` file. Then type the following command:
 
-
-```
-ln -s /Applications/Julia-{{stable_release_short}}.app/Contents/Resources/julia/bin/julia /usr/local/bin/julia
-```
+~~~
+<pre><code class="language-shell">ln -s /Applications/Julia-{{stable_release_short}}.app/Contents/Resources/julia/bin/julia /usr/local/bin/julia</code></pre>
```

Comment:
small nit, I think it should be

```
classs="lang-shell hljs"
```

to ensure highlight picks it up

Reply:
it probably doesn't matter I don't think it'll be highlighted anyway

Reply:
I believe that is the post-pre-rendering version, before that it is just `language-XXX`, no?

Reply:
yes

Reply:
Anyway, seems to have succesfully deployed and prerendered into:
```
<code class="shell hljs">wget https://julialang-s3.julialang.org/bin/linux/x64/1.5/julia-1.5.2-linux-x86_64.tar.gz
tar zxvf julia-1.5.2-linux-x86_64.tar.gz</code>
```